src/common/error.h

```diff
@@ -175,69 +175,61 @@ The seldom used "THROWP" variants allow an error to be thrown with a pointer to
 /***********************************************************************************************************************************
 Throw an error when a system call fails
 ***********************************************************************************************************************************/
+#define THROW_SYS_ERROR(errorType, message)                                                                                        \
+    errorInternalThrowSys(errno, &errorType, __FILE__, __func__, __LINE__, message)
+#define THROW_SYS_ERROR_FMT(errorType, ...)                                                                                        \
+    errorInternalThrowSysFmt(errno, &errorType, __FILE__, __func__, __LINE__, __VA_ARGS__)
+#define THROWP_SYS_ERROR(errorType, message)                                                                                       \
+    errorInternalThrowSys(errno, errorType, __FILE__, __func__, __LINE__, message)
+#define THROWP_SYS_ERROR_FMT(errorType, ...)                                                                                       \
+    errorInternalThrowSysFmt(errno, errorType, __FILE__, __func__, __LINE__, __VA_ARGS__)
+
+#define THROW_SYS_ERROR_CODE(errNo, errorType, message)                                                                            \
+    errorInternalThrowSys(errNo, &errorType, __FILE__, __func__, __LINE__, message)
+#define THROW_SYS_ERROR_CODE_FMT(errNo, errorType, ...)                                                                            \
+    errorInternalThrowSysFmt(errNo, &errorType, __FILE__, __func__, __LINE__, __VA_ARGS__)
+#define THROWP_SYS_ERROR_CODE(errNo, errorType, message)                                                                           \
+    errorInternalThrowSys(errNo, errorType, __FILE__, __func__, __LINE__, message)
+#define THROWP_SYS_ERROR_CODE_FMT(errNo, errorType, ...)                                                                           \
+    errorInternalThrowSysFmt(errNo, errorType, __FILE__, __func__, __LINE__, __VA_ARGS__)
+
 // When coverage testing define special versions of the macros that don't contain branches.  These macros are less efficient because
```

Comment:
I tripped over this first sentence" When coverage testing define special versions". Can you just put a comma after testing "When coverage testing, define special versions"?

Reply:
Committed at b7d8d61526e1e with comma added.